brainzutils/musicbrainz_db/release_group.py

```diff
@@ -163,3 +163,55 @@ def _get_release_groups_for_artist_query(db, artist_id, release_types):
         join(models.ArtistCreditName, models.ArtistCreditName.artist_credit_id == models.ReleaseGroup.artist_credit_id).\
         join(models.Artist, models.Artist.id == models.ArtistCreditName.artist_id).\
         filter(models.Artist.gid == artist_id).filter(models.ReleaseGroupPrimaryType.name.in_(release_types))
+
+
+def get_release_groups_for_label(label_id, release_types=None, limit=None, offset=None):
+    """Get all release groups linked to a label.
+
+    Args:
+        label_id (uuid): MBID of the label.
+        release_types (list): List of types of release groups to be fetched.
```

Comment:
this documentation should include a list of the types that are allowed.

I've asked the MusicBrainz developers how they validate and select this list of release types - because the types are defined in the database, so theoretically more types could be added without a schema change. Ideally this code would keep working with a new type. This means that we should actually check these types directly against the database for validation. I'll update this comment once I hear back from them about how it's done in MB

Reply:
Done